src/utils/helpers/date/date.js

```diff
@@ -34,6 +34,17 @@ const DateHelper = {
     return DateHelper._parseDate(value, options).isValid();
   },
 
+  /**
+   * Determins if value length is valid
+   *
+   * @param {String} value - value to validate
+   * @param {Integer} valueLength - the number that determines valid length
+   * @return {Boolean}
+   */
+  isValidLength: (value, validLength = 0) => {
+    return (value.length >= validLength);
```

Comment:
I think this could be updated to be just `>` rather than `>=` or you could update the getDisabledDays function to be 
```
if (minDate && DateHelper.isValidLength(minDate, 10) && DateHelper.isValidDate(minDate)) 
```
as the iso format (YYYY-MM-DD) is 10 digits.

Everything else looks good and I think it will still work as is but might be worth just tightening it up a little

Reply:
we have `allowEmptyValue` prop on date component, if it is set to `true` and then isValidLength would fail when having `>` instead of `>=`. I think what I proposed is best option, hope that makes sense. Please correct me if I am wrong!

Reply:
Ah yeah good point it does need to be `>=` I think it isn't essential to update  it to `DateHelper.isValidLength(minDate, 10)` then as the `isValidDate` catches anything that is only 9 chars long